node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/Rebalancer.java

```diff
@@ -99,6 +99,7 @@ private Move findBestMove(NodeList allNodes) {
         Move bestMove = Move.none;
         for (Node node : allNodes.nodeType(NodeType.tenant).state(Node.State.active)) {
             if (node.parentHostname().isEmpty()) continue;
+            if (node.allocation().map(allocation -> allocation.owner().instance().isTester()).orElse(false)) continue;
```

Comment:
Since this is just active nodes you can just say
if (node.allocation().get().owner().instance().isTester())) continue;

Reply:
Thanks.

Reply:
```suggestion
            if (node.allocation().get().owner().instance().isTester())) continue;
```